pkg/cmdlets/errors/errors.go

```diff
@@ -98,11 +98,21 @@ func ParseUserFacing(err error) (int, error) {
 	// unwrap exit code before we remove un-localized wrapped errors from err variable
 	code := errs.ParseExitCode(err)
 
+	// If there is a user facing error in the error stack we want to ensure
+	// that is it forwarded to the user.
+	uerr, ok := errs.IsUserFacing(err)
+	if ok {
+		logging.Debug("Returning user facing error, error stack: \n%s", errs.JoinMessage(err))
+		return code, &OutputError{uerr}
```

Comment:
I don't think OutputError produces the error string from `UserFacingError()`? 

Please make sure you run this code to ensure it produces the error message you expect.

Reply:
It does here: https://github.com/ActiveState/cli/blob/master/pkg/cmdlets/errors/errors.go. I've confirmed this from running the code.

Reply:
I don't understand, you linked to code on master which definitely would not be handling this new concept. Wrong link?

Reply:
Whoops, yes wrong link.

I've run the code and confirmed the output from `UserError()` is there. I believe it's done via the output mediator here: https://github.com/ActiveState/cli/blob/DX-2151/internal/output/mediator.go#L54